convert2rhel/checks.py

```diff
@@ -46,6 +46,18 @@
 KERNEL_REPO_VER_SPLIT_RE = re.compile(r"\W+")
 BAD_KERNEL_RELEASE_SUBSTRINGS = ("uek", "rt", "linode")
 
+GPG_PATH = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
```

Comment:
I would rename this to 

```suggestion
RPM_GPG_KEY_PATH = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
```

So it's more consistent with the actual file you want to use.

Reply:
ok cool! I'll go ahead and make that change, thank you for pointing that out!